config/apisix-versions.js

```diff
@@ -3,7 +3,7 @@
 /**
  * @type {Array<string>} version list
  */
-const versions = ['2.12', '2.13', '2.14', '2.15', '3.0', '3.1', '3.2', '3.3'];
+const versions = ['2.12', '2.13', '2.14', '2.15', '3.0', '3.1', '3.2', '3.3', '3.4'];
```

Comment:
Also removing 2.12 is better.

Reply:
OK, and where should I find the archived versions link for `2.12`, like:

![image](https://github.com/apache/apisix-website/assets/72343596/804136f8-2686-4036-a76e-598645590822)